packages/jest-jasmine2/src/treeProcessor.ts

```diff
@@ -65,10 +65,11 @@ export default function treeProcessor(options: Options): void {
   }
 
   function hasNoEnabledTest(node: TreeNode): boolean {
-    if (node.children) {
-      return node.children.every(hasNoEnabledTest);
-    }
-    return node.disabled || node.markedPending;
+    return (
+      node.disabled ||
+      node.markedPending ||
+      (!!node.children && node.children.every(hasNoEnabledTest))
```

Comment:
```suggestion
      node.children?.every(hasNoEnabledTest)
```

right?

Reply:
Yes. Will change this one as well. However since ts is prompting for a `boolean` return value, without double negation  it is throwing an error owing to a possible `undefined` return value. So it would be `node.children?.every(hasNoEnabledTest)`. Is it fine?

Reply:
`node.children?.every(hasNoEnabledTest) ?? false`

Reply:
Ok, I have done the changes